doofinder.php

```diff
@@ -1259,13 +1260,8 @@ private function indexApiInvokeReindexing()
         $api_key = Configuration::get('DF_API_KEY');
         $api = new DoofinderApiIndex($api_key, $region);
         $response = $api->invokeReindexing(Configuration::get('DF_INSTALLATION_ID'), $this->getProcessCallbackUrl());
-        if (empty($response)) {
-            $this->debug('Empty response from invoke reindexing');
-
-            return;
-        }
-        if (!empty($response['errors'])) {
-            $this->debug(json_encode($response['errors']));
+        if ($response['status'] != 200) {
```

Comment:
I think it would also be beneficial to check first `empty($response)`, just in case the inner post response is an empty array on some edge-cases.
```suggestion
        if (empty($response) || $response['status'] !== 200) {
```

Reply:
In this case we should always receive a JSON response from the plugins service, but if you think it could return an empty response in some cases, I'll apply your suggestion.